[Test only] include trt in cl docker

wushirong opened the pull request:
Summary:
[TEST ONLY]
This diff is testing whether install trt can help resolve header not found issue on another diff.

Test Plan: cl

Differential Revision: D32802182

malfet commented on .circleci/docker/common/install_tensorrt.sh:
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ -n "$TENSORRT" ]; then
```
```suggestion
if [ -n "$TENSORRT_VERSION" ]; then
```

malfet commented on .circleci/docker/common/install_tensorrt.sh:
```diff
@@ -0,0 +1,7 @@
+if [ -n "$TENSORRT" ]; then
+    python3 -m pip install --upgrade setuptools pip
+    python3 -m pip install nvidia-pyindex
+    python3 -m pip install nvidia-tensorrt==8.0.1.6 --extra-index-url https://pypi.ngc.nvidia.com
```
```suggestion
    python3 -m pip install nvidia-tensorrt==${TENSORRT_VERSION} --extra-index-url https://pypi.ngc.nvidia.com
```

malfet commented on .circleci/docker/ubuntu-cuda/Dockerfile:
```diff
@@ -67,6 +67,11 @@ ADD ./common/install_openssl.sh install_openssl.sh
 RUN bash ./install_openssl.sh
 
+# (optional) Install TensorRT
+ARG TENSORRT
```
```suggestion
ARG TENSORRT_VERSION
```

malfet commented on test/run_test.py:
```diff
@@ -85,7 +85,7 @@ def skip_test_p(name: str) -> bool:
         'custom_backend',
         'custom_operator',
-        'fx',        # executed by test_fx.py
+        'fx/',        # executed by test_fx.py
```
Can you please explain why those changes are needed?

wushirong replied:
> 

in dir caffe2/test, there are dir with name "fx", "fx_acc", "fx2trt", this change is to only block test under dir caffe2/test/fx/...

wushirong committed:
[Test only] include trt in cl docker (#69301)

Summary:
[TEST ONLY]
This diff is testing whether install trt can help resolve header not found issue on another diff.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/69301

Test Plan: cl

Differential Revision: D32802182

Pulled By: wushirong

fbshipit-source-id: 2b35f13cd2bccea0ae673da7eafe4a3fec8f3304
